test/jsonSchema.test.ts

```diff
@@ -113,6 +140,22 @@ const parquetSchema = ParquetSchema.fromJsonSchema({
       },
       additionalItems: false,
     },
+    time_field: {
+      type: 'object',
+      properties: {
+        value: {
+          type: 'number',
+        },
+        unit: {
+          type: 'string',
+          enum: ['MILLIS', 'MICROS', 'NANOS'], // Define enum for time units
+        },
+        isAdjustedToUTC: {
+          type: 'boolean',
+        },
+      },
+      additionalProperties: false,
+    },
```

Comment:
so is it how we expect logical type type be defined in schema? which ties to question about unit above